dashboard/app/controllers/featured_projects_controller.rb

```diff
@@ -1,23 +1,35 @@
 class FeaturedProjectsController < ApplicationController
+  # The first time a project is featured.
   def create
-    unless current_user.try(:permission?, UserPermission::PROJECT_VALIDATOR)
+    if current_user.try(:permission?, UserPermission::PROJECT_VALIDATOR)
```

Comment:
Oops. Was this actually backwards so anyone BUT project validators had access?

Do we want to return a different code (403 forbidden or 404 not found) for insufficient permissions. As it stands this will fail silently - the UI will receive a 200 and think it succeeded for non project validators.

On that note, if we're setting up auth, we should leverage `CanCan` (ability.rb and `authorize_resource`).